tests/accessibility/bolus_screen.tests.js

```diff
@@ -0,0 +1,31 @@
+var bolusScreenAccessibilityTests = (test) => {
```

Comment:
[nit] I notice some filenames use camelcase and others use snakecase?

Reply:
yeah good point - I will convert to one style